src/emqttd_access_control.erl

```diff
@@ -47,27 +38,24 @@
 
 -define(ACCESS_CONTROL_TAB, mqtt_access_control).
 
-%%%=============================================================================
-%%% API
-%%%=============================================================================
+-type password() :: undefined | binary().
+
+-record(state, {}).
+
+%%--------------------------------------------------------------------
+%% API
+%%--------------------------------------------------------------------
 
-%%------------------------------------------------------------------------------
-%% @doc Start access control server
-%% @end
-%%------------------------------------------------------------------------------
+%% @doc Start access control server.
 -spec start_link() -> {ok, pid()} | ignore | {error, any()}.
-start_link() ->
-    start_link(emqttd:env(access)).
+start_link() -> start_link(emqttd:env(access)).
 
 -spec start_link(Opts :: list()) -> {ok, pid()} | ignore | {error, any()}.
 start_link(Opts) ->
     gen_server:start_link({local, ?SERVER}, ?MODULE, [Opts], []).
 
-%%------------------------------------------------------------------------------
-%% @doc Authenticate MQTT Client
-%% @end
-%%------------------------------------------------------------------------------
--spec auth(mqtt_client(), undefined | binary()) -> ok | {error, string()}.
+%% @doc Authenticate MQTT Client.
+-spec auth(Client :: mqtt_client(), Password :: password()) -> ok | {error, any()}.
```

Comment:
Line 58 is too long: -spec auth(Client :: mqtt_client(), Password :: password()) -> ok | {error, any()}..